src/Controller/CRUDController.php

```diff
@@ -37,11 +39,21 @@
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 use Symfony\Component\Security\Csrf\CsrfToken;
 
+// BC for Symfony < 3.3 where this trait does not exist
+// NEXT_MAJOR: Remove the polyfill and inherit from Controller again
+require_once __DIR__.'/PolyfillControllerTrait.php';
```

Comment:
I'm not a big fan of vendor code copy. There is no another way? :confused:

Reply:
The other way I know of is dropping sf 2. This is ugly, but should work fairly well and is quite maintainable since we are not supposed to edit `PolyfillControllerTrait` at all

Reply:
I think it might be possible using Reflection: https://3v4l.org/ZEjh5

Problem is, `CRUDController` implements `__call` itself to fix Symfony 3.3 compatibility issues. Argh!

Reply:
@greg0ire You mean SF 3.2? I think we can. It's BC and SF 3.2 is not maintained except for security fixes which will BTW end at the end of the year.

Reply:
No, I mean sf 2.8. In sf 2, `render` is `public`, and can't be "overriden" with `__call` apparently.